Tests/OfflineTests/CombineSynchronizerOfflineTests.swift

```diff
@@ -218,14 +218,16 @@ class CombineSynchronizerOfflineTests: XCTestCase {
     }
 
     func testGetSaplingAddressSucceed() {
-        synchronizerMock.getSaplingAddressAccountIndexClosure = { accountIndex in
-            XCTAssertEqual(accountIndex, 3)
+        let testAccount = Account(3)
+        
+        synchronizerMock.getSaplingAddressAccountClosure = { account in
+            XCTAssertEqual(account, testAccount)
```

Comment:
```suggestion
        let testAccountIndex = Zip32AccountIndex(3)
        
        synchronizerMock.getSaplingAddressAccountIndexClosure = { accountIndex in
            XCTAssertEqual(accountIndex, testAccountIndex)
```
and similarly below.

Reply:
Updated